runtime/common/xwalk_runtime_features.cc

```diff
@@ -45,49 +98,34 @@ void XWalkRuntimeFeatures::Initialize(const base::CommandLine* cmd) {
     experimental_features_enabled_ = true;
   else
     experimental_features_enabled_ = false;
-  // Add new features here with the following parameters :
-  // - Name of the feature
-  // - Name of the command line switch which will be used after the
-  // --enable/--disable
-  // - Description of the feature
-  // - Status of the feature : experimental which is turned off by default or
-  // stable which is turned on by default
-  AddFeature("SysApps", "sysapps",
-             "Master switch for the SysApps category of APIs", Stable);
-  AddFeature("RawSocketsAPI", "raw-sockets",
-             "JavaScript support for using TCP and UDP sockets", Stable);
-  AddFeature("DeviceCapabilitiesAPI", "device-capabilities",
-             "JavaScript support for peeking at device capabilities", Stable);
-  AddFeature("StorageAPI", "storage",
-             "JavaScript support to file system beyond W3C spec", Stable);
+
+  std::vector<std::string> platform_features(
+      g_features, g_features + arraysize(g_features));
+  for (auto feature : g_all_features) {
+    if (std::find(platform_features.begin(),
+                  platform_features.end(),
+                  feature.name) != platform_features.end())
+      AddFeature(feature);
```

Comment:
can we just do
#if !defined(DISABLE_BUNDLED_EXTENSIONS)
#if defined(OS_ANDROID)
AddFeature("SysApps", "sysapps",  "Master switch for the SysApps category of APIs", Stable);
....
#else
AddFeature(....)
....
#endif
#endif 

so that we drop g_all_features & g_features.

LGTM otherwise